Gemfile

```diff
@@ -0,0 +1,11 @@
+source "https://rubygems.org"
```

Comment:
If this is intended to be distributed as a gem, the Gemfile should only delegate to the gemspec like this:

https://github.com/alphagov/govuk_schemas/blob/master/Gemfile